packages/orval/src/write-specs.ts

```diff
@@ -67,6 +67,8 @@ export const writeSpecs = async (
       ? '.ts'
       : output.fileExtension ?? '.ts';
 
+    const namingConvention = output.namingConvention ?? 'camelCase';
```

Comment:
The default value is determined when normalizing.

https://github.com/orval-labs/orval/blob/master/packages/orval/src/utils/options.ts#L148